regression-test/suites/vaults/forbid/forbid.groovy

```diff
@@ -0,0 +1,45 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("forbid_vault") {
+    if (enableStoragevault()) {
```

Comment:
we can also read the frontend config via MySQL

Reply:
Could you please provide some sample code? I think it would be significantly helpful since we can remove one config in the regression's conf file.

Reply:
It's not one config in FE, it's lied in FE's memory, currently we have no elegant way to find this config.